### PR TITLE
Update PokemonSprites with new properties.

### PR DESCRIPTION
**Pull request by Schinwinkwinsky**

### CODE REVIEW

File: PokeApiNet/Models/Pokemon.cs [hunk 1]
```diff
@@ -785,6 +785,988 @@ public class PokemonSprites
...
+            /// <summary>
+            /// Pókemon sprites for Generation III
+            /// </summary>
+            public class GenerationIIISprites
+            {
+                /// <summary>
+                /// Pókemon Esmerald sprites
+                /// </summary>
+                [JsonProperty("emerald")]
+                public EsmeraldSprites Esmerald { get; set; }
```
Comment (mtrdp642) [inline PokeApiNet/Models/Pokemon.cs:1194]: Here and elsewhere: `Esmerald` -> `Emerald`.

File: PokeApiNet/Models/Pokemon.cs [hunk 2]
```diff
@@ -785,6 +785,988 @@ public class PokemonSprites
...
+
+            /// <summary>
+            /// Pókemon sprites for Generation V
+            /// </summary>
+            public class GenerationVSprites
+            {
+                /// <summary>
+                /// Pókemon Black/White sprites
+                /// </summary>
+                [JsonProperty("black-write")]
```
Comment (mtrdp642) [inline PokeApiNet/Models/Pokemon.cs:1480]: `write` -> `white`

**Review by mtrdp642 — CHANGES_REQUESTED** [review]
Some small changes needed to get this fully working for all the added fields.

File: PokeApiNet/Models/Pokemon.cs [hunk 3]
```diff
@@ -785,6 +785,988 @@ public class PokemonSprites
...
+            /// <summary>
+            /// Pókemon sprites for Generation III
+            /// </summary>
+            public class GenerationIIISprites
+            {
+                /// <summary>
+                /// Pókemon Emerald sprites
+                /// </summary>
+                [JsonProperty("emerald")]
+                public EsmeraldSprites Emerald { get; set; }
```
Comment (mtrdp642) [inline PokeApiNet/Models/Pokemon.cs:1194]: Let's change the class from `EsmeraldSprites` to `EmeraldSprites`, please.

**Review by mtrdp642 — CHANGES_REQUESTED** [review]
Last change

File: PokeApiNet/Models/Pokemon.cs (hunk 3 above)
Comment (Schinwinkwinsky) [reply PokeApiNet/Models/Pokemon.cs]: Oh, man. I thought I had corrected them all. Sorry! Done.

**Review by mtrdp642 — APPROVED** [review]

**Pull request merged**